pkg/client/client.go

```diff
@@ -839,7 +842,7 @@ func (o *OptimizelyClient) getExperimentDecision(experimentKey string, userConte
 	}
 
 	options := &decide.Options{}
-	experimentDecision, err = o.DecisionService.GetExperimentDecision(decisionContext, userContext, options, decide.NewDecisionReasons(options))
+	experimentDecision, _, err = o.DecisionService.GetExperimentDecision(decisionContext, userContext, options)
```

Comment:
same across all code changes.

Reply:
yeah, but this function does not return reasons